operator/v1alpha1/types.go

```diff
@@ -150,3 +150,31 @@ type DelegatedAuthorization struct {
 	// disabled indicates that authorization should be disabled.  By default it will use delegated authorization.
 	Disabled bool `json:"disabled,omitempty"`
 }
+
+// StaticPodOperatorStatus is status for controllers that manage static pods.  There are different needs because individual
+// node status must be tracked.
+type StaticPodOperatorStatus struct {
+	OperatorStatus `json:",inline"`
+
+	// latestDeploymentGeneration is the deploymentID of the most recent deployment
+	LatestDeploymentGeneration int32 `json:"latestDeploymentGeneration"`
```

Comment:
latest in the sense of finished or in-progress? The relation to the values in NodeStatus should be clearer.

Reply:
> latest in the sense of finished or in-progress? The relation to the values in NodeStatus should be clearer.

Latest in the sense of most recent available level to install

Reply:
Difference to target?

Reply:
would add "Available" to the field name. `LatestAvailableDeploymentGeneration`. Long word. But makes it more clear, doesn't it?